static/app/views/alerts/incidentRules/ruleForm/index.tsx

```diff
@@ -788,7 +792,11 @@ class RuleFormContainer extends AsyncComponent<Props, State> {
 
 const StyledListItem = styled(ListItem)`
   margin: ${space(2)} 0 ${space(1)} 0;
+`;
+
+const ListItemTitle = styled('div')`
```

Comment:
Having a custom `ListItemTitle` component here makes my eye twitch a bit. Shouldn’t this problem be solved at the component level? Like Large/Small ListItem?